### PR TITLE
Irsa 570 tst add ztf mission control

### PR DESCRIPTION
**Pull request by wmiipac**
Adding ZTF mission in time series tool. The first review is done by EJ, Steve G and test group. Please approve this PR.

### CODE REVIEW

File: src/firefly/js/templates/lightcurve/LcConverterFactory.js [hunk 1]
```diff
@@ -122,7 +124,7 @@ const converters = {
         converterId: 'ptf',
         defaultImageCount: 5,
         defaultTimeCName: 'obsmjd',
-        defaultYCname: 'mag_autocorr',
+        defaultYCname: 'magerr_auto',
```
Comment (wmiipac) [inline src/firefly/js/templates/lightcurve/LcConverterFactory.js]: This must have being temporary  change when debug the Cname matching. should change back to "mag_autocorr"

File: src/firefly/config/app.prop [hunk 2]
```diff
@@ -5,10 +5,11 @@ visualize.fits.search.path=@visualize.fits.search.path@:@alerts.dir@:@work.direc
 irsa.gator.hostname=@irsa.gator.hostname@
 wise.ibe.host=@wise.ibe.host@
 twomass.ibe.host=@twomass.ibe.host@
+ztf.ibe.host=@ztf.ibe.host@
 atlas.ibe.host=@atlas.ibe.host@
 
 FireflyTools.Advertise= @FireflyTools.Advertise@
-ignore.auth=true
+ignore.auth=false
```
Comment (wmiipac) [inline src/firefly/config/app.prop]: change ignore.auth=true as general prop in firefly

File: src/firefly/js/templates/lightcurve/LcConverterFactory.js (hunk 1 above)
Comment (ejoliet) [reply src/firefly/js/templates/lightcurve/LcConverterFactory.js]: oh, you are right. Can you change it before merging?

File: config/app.config [hunk 3]
```diff
@@ -53,6 +53,7 @@ irsa.hips.masterUrl="https://irsa.ipac.caltech.edu/data/hips/list"
 
 wise.ibe.host       = "irsa.ipac.caltech.edu/ibe"
 twomass.ibe.host    = "irsa.ipac.caltech.edu/ibe"
+ztf.ibe.host        = "irsadev.ipac.caltech.edu/ibe"
```
Comment (loitly) [inline config/app.config]: This should be `irsa` and not `irsadev`.  `irsadev` can be used at the `dev` env block if needed.

File: src/firefly/html/demo/ffapi-highlevel-test.html [hunk 4]
```diff
@@ -59,7 +59,10 @@
...
-    <a href='javascript:launchLC()'>Launch WISE LcViewer</a>
+    <a href='javascript:launchTimeSeries()'>Launch Time Series</a>
+    <a href='javascript:firefly.getViewer(null, "ts.html").showTable(firefly.util.table.makeFileRequest(null, "http://web.ipac.caltech.edu/staff/ejoliet/demo/WISE-mep-2764p772_ac51-010204.tbl",null,{ tbl_id: "raw_table",META_INFO: {TSDatasetId: "wise"} }))'>Launch TS</a>
+    <a href='javascript:firefly.getViewer(null, "ts.html").showTable(firefly.util.table.makeFileRequest(null, "https://irsadev.ipac.caltech.edu:9028/cgi-bin/g2p/nph-g2p?dataid=source_id_mf&ws=/pubspace/70/12/b09751929eff1b93c90edca3eb30/g2p/32412&id=2948m228_ac51-042528",null,{ tbl_id: "raw_table",META_INFO: {TSDatasetId: "wise"} }))'>2948m228_ac51-042528</a>
+    <a href='javascript:firefly.getViewer(null, "ts.html").showTable(firefly.util.table.makeFileRequest(null, "https://irsadev.ipac.caltech.edu:9028/cgi-bin/g2p/nph-g2p?dataid=source_id_mf&ws=/pubspace/70/12/b09751929eff1b93c90edca3eb30/g2p/12875&id=2948m228_ac51-042528",null,{ tbl_id: "raw_table",META_INFO: {TSDatasetId: "wise"} }))' target='_blank'>Launch TS1</a>
```
Comment (loitly) [inline src/firefly/html/demo/ffapi-highlevel-test.html]: `9028` port should be removed before merging.

**Pull request merged**